src/index.html

```diff
@@ -13,6 +13,49 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+      <div class="stars stars--1">
+        <div class="stars__star star__yellow-star"></div>
```

Comment:
```suggestion
      <div class="stars stars--1">
        <div class="stars__star"></div>
```
try to rewrite this logic without using the additional classes

Reply:
looks like a redundant class at all